lab-edward/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "lab-edward",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "dependencies": {
+    "chai": "^3.5.0"
```

Comment:
You can get rid of this duplicate dependency; chai should just be a devDependency